src/locales/fr.json

```
@@ -5,7 +5,7 @@
     },
     "Delete": {
       "delete_note": "Supprimer",
-      "deleted": "La note à été supprimée",
+      "deleted": "La note à été placée dans la corbeille",
```

```suggestion
      "deleted": "La note a été placée dans la corbeille",
```

Je dirais même un truc du style: "La note a été placée dans la corbeille de votre Cozy Drive". Pour indiquer un peu la connexion entre Notes & Drive.